Assets/Global/Scripts/UIScripts/Healthbar.cs

```diff
@@ -45,8 +44,18 @@ public void UpdateMaxHealth() {
         // doing maxHealth / 2 because 1hp is a half heart
         float maxHealth = player.playerStatistic.MaxHealth.GetValue();
         HealthbarUnderlayTransform.sizeDelta = new Vector2(
-            originalWidth * (maxHealth / 2), 
+            originalWidth * (maxHealth / 2),
             HealthbarUnderlayTransform.sizeDelta.y
         );
     }
+
+    void UpdateCurrentHealthPermanent() {
```

Comment:
I am a bit confused. like i said the permenent functionality should not interfear with the rest of the application so it should not be hear.

But even if it should be here, Why are you then saving the stat even though nothing changed?
Or if something changed, why are you not saving it there?

This is not something that should now be fixed since it should be removed anyways. But yea i have no idea what is going on